File: Hypothesis Testing/Answer_Q5.py
import pandas as pd
import scipy
from scipy import stats
import statsmodels.api as sm
import numpy as np
sales=pd.read_csv('Fantaloons.csv')
from statsmodels.stats.proportion import proportions_ztest
tap1=sales.Weekdays.value_counts()
tap1
tap2=sales.Weekend.value_counts()
tap2
stats,pval = proportions_ztest(tap1, tap2,alternative='two-sided') 
print(pval) 
stats,pval = proportions_ztest(tap1, tap2,alternative='larger')
print(pval)    

File: Hypothesis Testing/Answer_Q5.R
sales=read.csv(file.choose())
attach(sales)
table1<-table(Weekdays)
table1
table2<-table(Weekend)
table2
prop.test(x=c(113,167),n=c(400,400),conf.level = 0.95,alternative = "two.sided")
prop.test(x=c(113,167),n=c(400,400),conf.level = 0.95,alternative = "greater")
